abci/types/application.go

```diff
@@ -31,6 +32,7 @@ type Application interface {
 	OfferSnapshot(context.Context, *RequestOfferSnapshot) (*ResponseOfferSnapshot, error)                // Offer a snapshot to the application
 	LoadSnapshotChunk(context.Context, *RequestLoadSnapshotChunk) (*ResponseLoadSnapshotChunk, error)    // Load a snapshot chunk
 	ApplySnapshotChunk(context.Context, *RequestApplySnapshotChunk) (*ResponseApplySnapshotChunk, error) // Apply a shapshot chunk
+	LoadLatest(context.Context, *RequestLoadLatest) (*ResponseLoadLatest, error)
```

Comment:
Can we add a small comment for what this interface do? Like the above ones?

Reply:
added